src/main/java/net/masterthought/cucumber/generators/AbstractPage.java

```diff
@@ -107,4 +116,20 @@ private void buildGeneralParameters() {
             }
         }
     }
+
+    /**
+     * Escapes all html that was provided in a reference before inserting it into a template.
+     */
+    private final static class EscapeHtmlReference implements ReferenceInsertionEventHandler {
+
+        @Override
+        public Object referenceInsert(String reference, Object value) {
+            if (value == null) {
```

Comment:
There is no specification so I copied that detail from the EscapeReference implementation but if you follow the data flow up to ASTReference.render you'll see that null is a plausible value.

Presumably when using

```
 $object.methodThatReturnsNull()
```